server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/DatafileManager.java

```diff
@@ -458,6 +457,7 @@ StoredTabletFile bringMajorCompactionOnline(Set<StoredTabletFile> oldDatafiles,
         filesInUseByScans, newFile, compactionId, dfv,
         tablet.getTabletServer().getClientAddressString(), lastLocation,
         tablet.getTabletServer().getLock());
+    tablet.setLastCompactionID(compactionId);
```

Comment:
I believe there is a related comment a few lines above that can probably be removed, about `known consistency issue between minor and major compactions`

Reply:
Actually that comment is still relevant.  The tablet's list of files are still being updated before the the metadata table update is made.  For minor compactions the tablets list of files is updated after the metadata update. The comment was referring to when the in memory list of files is updated.

Reply:
Oh my mistake.